src/components/Input/Input.js

```diff
@@ -0,0 +1,20 @@
+import { html } from '../../shared';
+import styles from './Input.module.css';
+
+/**
+ * @param {{minCharacters: number, maxCharacters: number, onChange: (e) => void }} props
```

Comment:
```suggestion
 * @param {{minCharacters: number, maxCharacters?: number, onChange: (e) => void }} props
```
`maxCharacters` ma domyślną wartość ale według typu jest wymagany